client-templates/gitlab/.env.sample

```diff
@@ -0,0 +1,23 @@
+# your unique broker identifier
+BROKER_TOKEN=
+
+# your personal token to your bitbucket server account
+GITLAB_TOKEN=
+
+# the host where your Gitlab Server is running, excluding scheme.
+# for gitlab.yourdomain.com
+# this should be "gitlab.yourdomain.com"
+GITLAB=
```

Comment:
We can probably just use `${GITLAB}` in code, and remove the duplication of `GITLAB_API` from the settings?